### DISCUSSION TITLE
Ip Pools Postal Smtp

### DISCUSSION THREAD

**Discussion by fav22** [postalserver/postal#1814] [Help with using Postal]
Hello , hope you all doing well !
I want to ask if someone here can help me about how to configure Ip pools with Postal smtp please ? 
Nb: i already tried to add additionnal ip on Postal dashboard but nothing happen , all mail goes through the principal Ip adress😢

Thank you ,

**Comment by willpower232** [comment]
Have you definitely 
- made sure the IP addresses are available on your server
- updated the config file
- restarted all of postal

**Reply by fav22** [reply]
Hello , sorry for the late reply ! How to add add those additional ip adress on server please ? Thank you for you reply

**Reply by willpower232** [reply]
You need to ask your server provider how to have multiple IP addresses on it

**Reply by fav22** [reply]
Okay , thank you !
